CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## 💡 Enhancements 💡
 
+- `coralogixexporter`: Add Coralogix Exporter (7383)
```

Comment:
You can unify both entries in one. Also, it's missing the "#" in front of the PR number.